oak_derive/Cargo.toml

```diff
@@ -14,4 +14,4 @@ quote = "1.0"
 proc-macro2 = "1.0"
 
 [dev-dependencies]
```

Comment:
This file should also have a [workspace] tag at the top, seeing that it is now a top-level workspace crate.

Reply:
Done